src/examples/avatar/AvatarType.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { Avatar } from '@zendeskgarden/react-avatars';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { ReactComponent as UserStrokeIcon } from '@zendeskgarden/svg-icons/src/16/user-solo-stroke.svg';
+
+const StyledCol = styled(Col)`
+  margin-bottom: ${p => p.theme.space.sm};
+  text-align: center;
+`;
+
+const Example = () => {
+  const iconColor = DEFAULT_THEME.palette.grey[600];
+
+  return (
+    <Grid>
+      <Row alignItems="center">
+        <StyledCol>
+          <Avatar backgroundColor={iconColor} size="large">
```

Comment:
Should these be the default size (remove `size` prop)?

Reply:
Yes I think so. I went ahead and removed the `size` prop.